docs/first_steps/first_steps.rst

```diff
@@ -160,6 +160,12 @@ The user can customize several aspects of Shapes Demo operation:
 
     + Push the *Start* button in order to resume Shapes Demo operation.
 
+- **Shared Memory Transport:** Fast DDS transport using Shared Memory Segment for Participants running in the same
```

Comment:
I am not sure why but when generating the documentation locally, this option is shown in a blue box. Can you check in your side if it is something happening only on my side?